tests/test_bp_save_optimization.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import nose
+
+import angr
+import ailment
+
+test_location = str(
+    os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), '../../binaries/tests'))
+
+
+def test_bp_save_fauxware(arch):
+    p = angr.Project(
+        test_location + '/{}/fauxware'.format(arch), auto_load_libs=False)
```

Comment:
Nitpick: Can you use `os.path.join()` instead?

Reply:
Yup!